### ISSUE TITLE
[Feature]: Call the callback on PromptUserForPushPermissions for Android SDK 33+

### ISSUE THREAD

**Issue by indiecore** [Appboy/appboy-unity-sdk#99]
### What problem are you facing?

Currently when you call `AppboyBinding.PromptUserForPushPermissions` on Android the callback simply never happens. This means you cannot wait on the user to make a choice and you can't check what the user's opt in state is.

### Workarounds

Currently I just immediately call back on my own stack. This means that the app can't actually wait till the user chooses a push notification option. I also means I must assume that the user approved push notifications even if they didn't actually.

### Ideal Solution

The callback from `PromptUserForPushPermissions` works identically to iOS where it will return `true` if the user accepted push notifications and `false` if they did not.

**Comment by indiecore** [comment]
1. Pixel 7, Android 13 (SDK 33)

2.

```
    public void openNativeNotificationPrompt()
    {
        Debug.Log("[K]: Calling Push Prompt");
        AppboyBinding.PromptUserForPushPermissions(false, (response) => {
            Debug.Log($"[K]: Braze response: {response}");
        });
    }
```

Calling this function results in:

```
01-18 15:56:18.469   894   894 I WifiHAL : Creating message to get link statistics; iface = 47
01-18 15:56:18.572   894   894 I WifiHAL : In GetLinkStatsCommand::handleResponse
01-18 15:56:18.616  1391  1738 D ConnectivityService: NetReassign [no changes] [c 3] [a 3] [i 6]
01-18 15:56:18.737  1049  1220 D CHRE    : @ 12934.697: [ActivityPlatform] type 4, confidence 95
01-18 15:56:18.783  1688  1700 I native  : I0000 00:00:1674075378.782587    1700 tf_lite_classifier.cc:383] Event stream classified as kNone
01-18 15:56:18.792  9801  9900 I Unity   : [K]: Calling Push Prompt
01-18 15:56:18.792  9801  9900 I Unity   : ChickletTPIRConsolePushNotifications:openNativeNotificationPrompt()
01-18 15:56:18.792  9801  9900 I Unity   : UnityEngine.Events.UnityEvent:Invoke()
01-18 15:56:18.792  9801  9900 I Unity   : AnimButton:pointerClick(BaseEventData)
01-18 15:56:18.792  9801  9900 I Unity   : UnityEngine.Events.UnityEvent`1:Invoke(T0)
01-18 15:56:18.792  9801  9900 I Unity   : UnityEngine.EventSystems.EventTrigger:Execute(EventTriggerType, BaseEventData)
01-18 15:56:18.792  9801  9900 I Unity   : UnityEngine.EventSystems.ExecuteEvents:Execute(GameObject, BaseEventData, EventFunction`1)
01-18 15:56:18.792  9801  9900 I Unity   : UnityEngine.EventSystems.StandaloneInputModule:ProcessTouchPress(PointerEventData, Boolean, Boolean)
01-18 15:56:18.792  9801  9900 I Unity   : UnityEngine.EventSystems.StandaloneInputModule:ProcessTouchEvents()
01-18 15:56:18.792  9801  9900 I Unity   : UnityEngine.EventSystems.StandaloneInputModule:Process()
01-18 15:56:18.792  9801  9900 I Unity   : 
01-18 15:56:18.792  9801  9900 V Braze v23.3.0 .PermissionUtils: Attempting to execute requestPushPermissionPrompt()
01-18 15:56:18.793  9801  9900 I Braze v23.3.0 .PermissionUtils: Notification permission already granted, doing nothing.
01-18 15:56:18.793  9801  9900 D Braze v23.3.0 .PermissionUtils: Permission prompt would not display, not attempting to request push permission prompt.
```

If the push notification prompt was already accepted (in the same situation iOS would respond with either true or false, this returns neither).

If the push has not previous been accepted we have longer reponse

```
01-18 16:06:16.860 11665 11757 I Unity   : [K]: Calling Push Prompt
01-18 16:06:16.860 11665 11757 I Unity   : ChickletTPIRConsolePushNotifications:openNativeNotificationPrompt()
01-18 16:06:16.860 11665 11757 I Unity   : UnityEngine.Events.UnityEvent:Invoke()
01-18 16:06:16.860 11665 11757 I Unity   : AnimButton:pointerClick(BaseEventData)
01-18 16:06:16.860 11665 11757 I Unity   : UnityEngine.Events.UnityEvent`1:Invoke(T0)
01-18 16:06:16.860 11665 11757 I Unity   : UnityEngine.EventSystems.EventTrigger:Execute(EventTriggerType, BaseEventData)
01-18 16:06:16.860 11665 11757 I Unity   : UnityEngine.EventSystems.ExecuteEvents:Execute(GameObject, BaseEventData, EventFunction`1)
01-18 16:06:16.860 11665 11757 I Unity   : UnityEngine.EventSystems.StandaloneInputModule:ProcessTouchPress(PointerEventData, Boolean, Boolean)
01-18 16:06:16.860 11665 11757 I Unity   : UnityEngine.EventSystems.StandaloneInputModule:ProcessTouchEvents()
01-18 16:06:16.860 11665 11757 I Unity   : UnityEngine.EventSystems.StandaloneInputModule:Process()
01-18 16:06:16.860 11665 11757 I Unity   : 
01-18 16:06:16.861 11665 11757 V Braze v23.3.0 .PermissionUtils: Attempting to execute requestPushPermissionPrompt()
01-18 16:06:16.865 11665 11757 V Braze v23.3.0 .PermissionUtils: Push Prompt can be shown on this device, within a reasonable confidence.
01-18 16:06:16.865 11665 11757 D Braze v23.3.0 .PermissionUtils: Incrementing permission request counter to 1.
01-18 16:06:16.865 11665 11757 V Braze v23.3.0 .PermissionUtils: Requesting push permission from system.
--------- beginning of system
01-18 16:06:16.867  1391  1757 I ActivityTaskManager: START u0 {act=android.content.pm.action.REQUEST_PERMISSIONS pkg=com.google.android.permissioncontroller cmp=com.google.android.permissioncontroller/com.android.permissioncontroller.permission.ui.GrantPermissionsActivity (has extras)} from uid 10322
01-18 16:06:16.881 11665 11665 D Braze v23.3.0 .UnityConfigurationProvider: Using resources value for key: 'com_braze_inapp_show_inapp_messages_automatically' and value: 'true'
01-18 16:06:16.881 11665 11665 V Braze v23.3.0 .BrazeInAppMessageManager: Unregistering InAppMessageManager from activity: com.appboy.unity.AppboyUnityPlayerActivity
01-18 16:06:16.883 11665 11665 D UnityAds: com.unity3d.services.core.webview.WebViewApp.invokeJavascriptMethod() (line:2) :: Invoking javascript: javascript:window.nativebridge.handleEvent(["LIFECYCLE","PAUSED","com.appboy.unity.AppboyUnityPlayerActivity"]);
01-18 16:06:16.886 11665 12211 V Adjust  : Foreground timer suspended with 27.6 seconds left
01-18 16:06:16.886 11665 12211 V Adjust  : Subsession end
01-18 16:06:16.886 11665 12211 D Adjust  : Wrote Activity state: ec:0 sc:1 ssc:1 sl:32.4 ts:32.4 la:11:12:13 uuid:eef4fcb3-2122-4366-9f56-ffbcac4cd87e
01-18 16:06:16.888 11665 11757 D games.[game name]: PlayerBase::stop() from IPlayer
01-18 16:06:16.888 11665 11757 D AudioTrack: stop(48): called with 1787648 frames delivered
01-18 16:06:16.889  2506  5875 I NearbyFastPair: TriangleSessionTracker: flush media count=0, media duration=27s, call count=0 [CONTEXT service_id=265 ]
01-18 16:06:16.890  3402  5993 I AiAiAdaptiveAudio: updateState: enabledInSettings=false, playbackState=PlaybackState{isPlaying=false, contentType=UNSET, isMusicContent=false}
01-18 16:06:16.890 15809 15809 D BoundBrokerSvc: onBind: Intent { act=com.google.android.gms.measurement.START pkg=com.google.android.gms }
01-18 16:06:16.890  3402  5993 I AiAiAdaptiveAudio: stopMonitoring
01-18 16:06:16.890 15809 15809 D BoundBrokerSvc: Loading bound service for intent: Intent { act=com.google.android.gms.measurement.START pkg=com.google.android.gms }
01-18 16:06:16.890  3402  5993 I AiAiAdaptiveAudio: Updating calibration state: IDLE -> IDLE
01-18 16:06:16.897   862   862 D usf_sensor_hal: accelerometer: Enter Batch. period = 50000000, latency = 0.
01-18 16:06:16.898  1048  1048 D AOC     : A3:MSG: usf_sensor.cc, 1872: USF: Received reconfig sampling request for LSM6DSV Accelerometer.
01-18 16:06:16.898   862   862 D usf_sensor_hal: IMU temperature: Enter Batch. period = 533334000, latency = 0.
01-18 16:06:16.899  1391  3496 W ActivityTaskManager: Tried to set launchTime (0) < mLastActivityLaunchTime (12235313)
01-18 16:06:16.910  3420  4492 I AssistantForeground: (REDACTED) Get launcher package: %s
01-18 16:06:16.918   894   894 I WifiHAL : In GetLinkStatsCommand::handleResponse
01-18 16:06:16.921  1391  4257 D CoreBackPreview: Window{9601858 u0 com.google.android.permissioncontroller/com.android.permissioncontroller.permission.ui.GrantPermissionsActivity}: Setting back callback OnBackInvokedCallbackInfo{mCallback=android.window.IOnBackInvokedCallback$Stub$Proxy@ccdfe17, mPriority=0}
01-18 16:06:16.938  1391  1558 I ActivityTaskManager: Displayed com.google.android.permissioncontroller/com.android.permissioncontroller.permission.ui.GrantPermissionsActivity: +65ms
01-18 16:06:16.943  1391  1738 D ConnectivityService: NetReassign [no changes] [c 0] [a 1] [i 1]
01-18 16:06:16.945   458   458 I hwservicemanager: getTransport: Cannot find entry android.hardware.graphics.allocator@4.0::IAllocator/default in either framework or device VINTF manifest.
01-18 16:06:16.946   457   457 I servicemanager: Found android.hardware.graphics.allocator.IAllocator/default in device VINTF manifest.
01-18 16:06:17.003   825   871 D audio_hw_output_stream: update stream 3 active 0 gain 0.000000
01-18 16:06:17.121  1049  1220 D CHRE    : @ 13533.082: [ActivityPlatform] type 4, confidence 55
01-18 16:06:18.522  1981  2942 D GRIL-S  : marginalCoverageInfoChanged
01-18 16:06:18.524  2876  2894 D ConnectivityMonitorStateMachine: [onMarginalCoverageInfoChanged] {subId=1} coverageInfo= MarginalCoverageInfo { slotId=0 inMarginalCoverage=false marginalCoverageTriggerEvent=0}
01-18 16:06:18.525  2876  8439 D ConnectivityMonitorStateMachine: [CampedOnLteOrNr] {subId=1} processMessage { when=-1ms what=544775 obj=MarginalCoverageInfo { slotId=0 inMarginalCoverage=false marginalCoverageTriggerEvent=0} target=afr }
01-18 16:06:18.525  2876  8439 D ConnectivityMonitorStateMachine: [InService] {subId=1} processMessage { when=-1ms what=544775 obj=MarginalCoverageInfo { slotId=0 inMarginalCoverage=false marginalCoverageTriggerEvent=0} target=afr }
01-18 16:06:18.525  2876  8439 I MarginalCoverageInfoHandler: MarginalCoverageInfo =inMarginalCoverage: false, Trigger Event: 0
01-18 16:06:18.696  1048  1048 D AOC     : A3:MSG: window_orientation.c, 697: [WO] Proposed rotation: 1, flat angle threshold: 40
01-18 16:06:18.696  1048  1048 D AOC     : A3:MSG: window_orientation.c, 1105: [WO] orientation angle 83, orientation 1
01-18 16:06:18.696  1048  1048 D AOC     : A3:MSG: window_orientation.c, 430: [WO] Gyro angle integrated: 4.721828
01-18 16:06:18.696  1048  1048 D AOC     : A3:MSG: window_orientation.c, 430: [WO] Gyro angle integrated: 4.721828
01-18 16:06:18.696  1048  1048 D AOC     : A3:MSG: window_orientation.c, 514: [WO] Rejected by flat_time. Last flat detected 0 ms ago. Tilt angle: 40
01-18 16:06:18.961   528 18057 D display : setActiveConfigWithConstraints:: PrimaryDisplay config(35) test(0)
01-18 16:06:19.949   894   894 I WifiHAL : Creating message to get link statistics; iface = 47
01-18 16:06:19.966   528 18057 D display : setActiveConfigWithConstraints:: PrimaryDisplay config(36) test(0)
01-18 16:06:19.993  1048  1048 D AOC     : F1:MSG: controller.cc, 273: AudioOutCtrl: cmd ID: 0x00c9, tag: 0x9c [cntr = 161]
01-18 16:06:19.993  1048  1048 D AOC     : F1:ERR: controller_audio_output.cc, 1497: Entrypoint 23 cannot be disabled
01-18 16:06:19.993  1048  1048 D AOC     : F1:ERR: controller.cc, 808: AudioOutCtrl: ipc: audio_output_co, cmd ID: 0x00c9, tag: 0x9c, rc: -1
01-18 16:06:19.994   825   871 D audio_hw: shutdown: raw-playback
01-18 16:06:19.995   825   871 D audio_hw_aoc_route: speaker-post 0
01-18 16:06:19.996  1048  1048 D AOC     : F1:MSG: controller.cc, 273: AudioOutCtrl: cmd ID: 0x010f, tag: 0x9d [cntr = 162]
01-18 16:06:19.996   825   871 D audio_hw_aoc_route: raw-playbackP 0
01-18 16:06:19.996   825   871 D audio_hw_aoc: aoc_get_hw_intf_ctrl : HW_INTF : 9
01-18 16:06:19.996  1048  1048 D AOC     : F1:Unregistered Source 9 from Sink 0 
01-18 16:06:19.996   825   871 W audio_hw_35l41: cs35l41_amp_common_event: ret: 0, event: 3, state: 3, device: 4
01-18 16:06:19.996  1048  1048 D AOC     : F1:AT Map:0 (LL:0), 0 EPs active, Power: Yes, Config: No
01-18 16:06:19.996  1048  1048 D AOC     : F1:Mixer 0 configuration changed: (0)
01-18 16:06:19.996  1048  1048 D AOC     : F1:Sink 0 Configuration changed: ULL (disabled)
01-18 16:06:19.997  1048  1048 D AOC     : F1:MSG: controller.cc, 811: AudioOutCtrl: ipc: audio_output_co, cmd ID: 0x010f, tag: 0x9d, rc: 0
01-18 16:06:19.997  1048  1048 D AOC     : H0:Mixer AMixSPKR: 480 samples (0000/0000) (OFF)
01-18 16:06:19.997   825  1326 D audio_hw_battery_adaptive_audio_control: ProcessPlaybackTunerControl: update playback tuner 2
01-18 16:06:19.997   825  1326 D audio_hw_waves_cust_action: UpdateThrottleState: 1->2 T
01-18 16:06:19.997   825  1326 D audio_hw_waves_cust_action: UpdateThrottleState: update tuning!
01-18 16:06:20.002   825   871 D audio_hw_aoc_route: speaker 0
01-18 16:06:20.003   825   871 W audio_hw_35l41: cs35l41_amp_common_event: ret: 0, event: 4, state: 2, device: 4
01-18 16:06:20.003   825   871 D audio_hw_soundtrigger: st_comm_aud_event_monitor:codec dev:4 inactive
01-18 16:06:20.003   825   871 I audio_hw_aoc: Mode Ambient is already selected
01-18 16:06:20.060  1688  1700 I native  : I0000 00:00:1674075980.059799    1700 tf_lite_classifier.cc:383] Event stream classified as kNone
01-18 16:06:20.072  2700  2700 V GrantPermissionsViewModel: Logged buttons presented and clicked permissionGroupName=android.permission-group.NOTIFICATIONS uid=10322 selectedLocations=0 package=com.[game.name] presentedButtons=9 clickedButton=1 sessionId=-1333980762913239740 targetSdk=33
01-18 16:06:20.074   894   894 I WifiHAL : In GetLinkStatsCommand::handleResponse
01-18 16:06:20.081  2700  2700 V GrantPermissionsViewModel: Permission grant result requestId=-1333980762913239740 callingUid=10322 callingPackage=com.[game.name] permission=android.permission.POST_NOTIFICATIONS isImplicit=false result=4
01-18 16:06:20.081   899 20384 D AF::TrackHandle: OpPlayAudio: track:66 usage:1 not muted
01-18 16:06:20.110  1391  1738 D ConnectivityService: NetReassign [no changes] [c 0] [a 1] [i 1]
01-18 16:06:20.140  1391  4602 D CompatibilityChangeReporter: Compat change id reported: 181136395; UID 10322; state: DISABLED
01-18 16:06:20.140  1391  4602 D CompatibilityChangeReporter: Compat change id reported: 174042936; UID 10322; state: DISABLED
01-18 16:06:20.146 30170 30183 I Finsky  : [41] hmy.a(16): com.[game.name]: Account from first account - [NzhnRH0jN9mDeNLqSIZ5cLwtxliXPODW7aNaLC0x7dw]
01-18 16:06:20.146 30170 30183 I Finsky  : [41] hmy.a(61): Billing preferred account via installer for com.[game.name]: [NzhnRH0jN9mDeNLqSIZ5cLwtxliXPODW7aNaLC0x7dw]
01-18 16:06:20.148 11665 11665 D UnityAds: com.unity3d.services.core.webview.WebViewApp.invokeJavascriptMethod() (line:2) :: Invoking javascript: javascript:window.nativebridge.handleEvent(["LIFECYCLE","RESUMED","com.appboy.unity.AppboyUnityPlayerActivity"]);
01-18 16:06:20.153 11665 11665 D Braze v23.3.0 .UnityConfigurationProvider: Using resources value for key: 'com_braze_inapp_show_inapp_messages_automatically' and value: 'true'
01-18 16:06:20.153 11665 11665 V Braze v23.3.0 .BrazeInAppMessageManager: Registering InAppMessageManager with activity: com.appboy.unity.AppboyUnityPlayerActivity
01-18 16:06:20.153 11665 11665 D Braze v23.3.0 .BrazeInAppMessageManager: Removing existing in-app message event subscriber before subscribing a new one.
01-18 16:06:20.154 11665 11665 D Braze v23.3.0 .BrazeInAppMessageManager: Subscribing in-app message event subscriber
01-18 16:06:20.154 11665 11665 V Braze v23.3.0 .BrazeInAppMessageManager: Removing existing sdk data wipe event subscriber before subscribing a new one.
01-18 16:06:20.154 11665 11665 V Braze v23.3.0 .BrazeInAppMessageManager: Subscribing sdk data wipe subscriber
01-18 16:06:20.168  1391  3496 D CompatibilityChangeReporter: Compat change id reported: 214016041; UID 10322; state: ENABLED
01-18 16:06:20.169 11665 11757 I Unity   : Checking for deep links
01-18 16:06:20.169 11665 11757 I Unity   : Helpers:LogFormat(LogType, Object, String, String)
01-18 16:06:20.169 11665 11757 I Unity   : RunParameters:InitDeepLinkParams()
01-18 16:06:20.169 11665 11757 I Unity   : 
01-18 16:06:20.169 11665 11757 V com.facebook.unity.FB: GetAppLink({"callback_id":"4"})
01-18 16:06:20.169 11665 11757 V com.facebook.unity.FB: sending to Unity OnGetAppLinkComplete({"callback_id":"4","did_complete":true})
01-18 16:06:20.170  4289  4289 I GoogleInputMethodService: GoogleInputMethodService.onFinishInput():3227 
01-18 16:06:20.171  4289  4289 I GoogleInputMethodService: GoogleInputMethodService.updateDeviceLockedStatus():2087 repeatCheckTimes = 0, unlocked = true
01-18 16:06:20.171  4289  4289 I GoogleInputMethodService: GoogleInputMethodService.onStartInput():1877 onStartInput(EditorInfo{inputType=0x0(NULL) imeOptions=0x0 privateImeOptions=null actionName=UNSPECIFIED actionLabel=null actionId=0 initialSelStart=-1 initialSelEnd=-1 initialCapsMode=0x0 hintText=null label=null packageName=com.[game.name] fieldId=2131362283 fieldName=null extras=null}, false)
01-18 16:06:20.171  4289  4289 I GoogleInputMethodService: GoogleInputMethodService.shouldHideHeaderOnInitialState():4008 ShouldHideHeaderOnInitialState = false
01-18 16:06:20.171  4289  4289 I GoogleInputMethodService: GoogleInputMethodService.updateDeviceLockedStatus():2087 repeatCheckTimes = 2, unlocked = true
01-18 16:06:20.172   862   862 D usf_sensor_hal: accelerometer: Enter Flush
01-18 16:06:20.175   862   862 D usf_sensor_hal: accelerometer: Enter Batch. period = 20000000, latency = 0.
01-18 16:06:20.177  1048  1048 D AOC     : A3:MSG: usf_sensor.cc, 1872: USF: Received reconfig sampling request for LSM6DSV Accelerometer.
01-18 16:06:20.177   862   862 D usf_sensor_hal: IMU temperature: Enter Batch. period = 533334000, latency = 0.
01-18 16:06:20.179 11665 12211 V Adjust  : Foreground timer starting
01-18 16:06:20.179 11665 12211 V Adjust  : Subsession start
01-18 16:06:20.179 11665 12211 V Adjust  : Started subsession 2 of session 1
01-18 16:06:20.180 11665 12211 D Adjust  : Wrote Activity state: ec:0 sc:1 ssc:2 sl:35.7 ts:32.4 la:11:12:13 uuid:eef4fcb3-2122-4366-9f56-ffbcac4cd87e
01-18 16:06:20.180 11665 12211 D Adjust  : Should not try to read Install referrer
01-18 16:06:20.181  3420  4288 I AssistantForeground: (REDACTED) Get launcher package: %s
01-18 16:06:20.181  1391  3496 D CoreBackPreview: Window{9601858 u0 com.google.android.permissioncontroller/com.android.permissioncontroller.permission.ui.GrantPermissionsActivity}: Setting back callback null
01-18 16:06:20.181  1391  1757 W InputManager-JNI: Input channel object '9601858 com.google.android.permissioncontroller/com.android.permissioncontroller.permission.ui.GrantPermissionsActivity (client)' was disposed without first being removed with the input manager!
01-18 16:06:20.184 30170  7418 I Finsky  : [254] hmy.a(16): com.[game.name]: Account from first account - [NzhnRH0jN9mDeNLqSIZ5cLwtxliXPODW7aNaLC0x7dw]
01-18 16:06:20.184 30170 30183 I Finsky  : [41] hmy.a(16): com.[game.name]: Account from first account - [NzhnRH0jN9mDeNLqSIZ5cLwtxliXPODW7aNaLC0x7dw]
01-18 16:06:20.184 30170  7418 I Finsky  : [254] hmy.a(61): Billing preferred account via installer for com.[game.name]: [NzhnRH0jN9mDeNLqSIZ5cLwtxliXPODW7aNaLC0x7dw]
01-18 16:06:20.184 30170 30183 I Finsky  : [41] hmy.a(61): Billing preferred account via installer for com.[game.name]: [NzhnRH0jN9mDeNLqSIZ5cLwtxliXPODW7aNaLC0x7dw]
01-18 16:06:20.187   825   871 D audio_hw_output_stream: update stream 3 active 1 gain 0.077625
01-18 16:06:20.187   825 14215 D audio_hw: prepare raw-playback
01-18 16:06:20.187   825 14215 D audio_hw: raw-playback:
01-18 16:06:20.187   825 14215 D audio_hw: rx:
01-18 16:06:20.187   825 14215 D audio_hw:   #0: OUT_SPEAKER_BE_CFG 0
01-18 16:06:20.187   825 14215 D audio_hw_aoc: aoc_get_hw_intf_ctrl : HW_INTF : 9
01-18 16:06:20.187   825 14215 D audio_hw_aoc: aoc_get_hw_intf_ctrl : HW_INTF : 9
01-18 16:06:20.187   825 14215 W audio_hw_35l41: cs35l41_amp_common_event: ret: 0, event: 1, state: 2, device: 4
01-18 16:06:20.187   825 14215 D audio_route: Apply path: speaker
01-18 16:06:20.188  3402  5993 I AiAiAdaptiveAudio: updateState: enabledInSettings=false, playbackState=PlaybackState{isPlaying=true, contentType=UNCHECKED_CONTENT_TYPE, isMusicContent=false}
01-18 16:06:20.188  3402  5993 I AiAiAdaptiveAudio: stopMonitoring
01-18 16:06:20.188  3402  5993 I AiAiAdaptiveAudio: Updating calibration state: IDLE -> IDLE
01-18 16:06:20.189 11665 11757 I Unity   : [Deep Links]: Deep link is 
01-18 16:06:20.189 11665 11757 I Unity   : Helpers:LogFormat(LogType, Object, String, String)
01-18 16:06:20.189 11665 11757 I Unity   : RunParameters:GotAppLink(IAppLinkResult)
01-18 16:06:20.189 11665 11757 I Unity   : Facebook.Unity.CallbackManager:TryCallCallback(Object, IResult)
01-18 16:06:20.189 11665 11757 I Unity   : Facebook.Unity.CallbackManager:CallCallback(Object, IResult)
01-18 16:06:20.189 11665 11757 I Unity   : Facebook.Unity.CallbackManager:OnFacebookResponse(IInternalResult)
01-18 16:06:20.189 11665 11757 I Unity   : 
01-18 16:06:20.189 11665 11757 I Unity   : Getting deferred deep link
01-18 16:06:20.189 11665 11757 I Unity   : Helpers:LogFormat(LogType, Object, String, String)
01-18 16:06:20.189 11665 11757 I Unity   : RunParameters:GotAppLink(IAppLinkResult)
01-18 16:06:20.189 11665 11757 I Unity   : Facebook.Unity.CallbackManager:TryCallCallback(Object, IResult)
01-18 16:06:20.189 11665 11757 I Unity   : Facebook.Unity.CallbackManager:CallCallback(Object, IResult)
01-18 16:06:20.189 11665 11757 I Unity   : Facebook.Unity.CallbackManager:OnFacebookResponse(IInternalResult)
01-18 16:06:20.189 11665 11757 I Unity   : 
01-18 16:06:20.189 11665 11757 V com.facebook.unity.FB: FetchDeferredAppLinkData({"callback_id":"5"})
01-18 16:06:20.190 11665 11665 D MediaRouter: onRestoreRoute() : route=RouteInfo{ name=Phone, description=null, status=null, category=RouteCategory{ name=System types=ROUTE_TYPE_LIVE_AUDIO ROUTE_TYPE_LIVE_VIDEO  groupable=false }, supportedTypes=ROUTE_TYPE_LIVE_AUDIO ROUTE_TYPE_LIVE_VIDEO , presentationDisplay=null }
01-18 16:06:20.190 11665 11665 V MediaRouter: Selecting route: RouteInfo{ name=Phone, description=null, status=null, category=RouteCategory{ name=System types=ROUTE_TYPE_LIVE_AUDIO ROUTE_TYPE_LIVE_VIDEO  groupable=false }, supportedTypes=ROUTE_TYPE_LIVE_AUDIO ROUTE_TYPE_LIVE_VIDEO , presentationDisplay=null }
01-18 16:06:20.194   825 14215 D audio_hw_aoc_route: speaker 1
01-18 16:06:20.194   825 14215 D audio_route: Apply path: hostless-ulC spk-vi
01-18 16:06:20.195   825 14215 W audio_hw_35l41: cs35l41_amp_common_event: ret: 0, event: 2, state: 3, device: 4
01-18 16:06:20.195   825 14215 D audio_hw_soundtrigger: st_comm_aud_event_monitor:codec dev:4 active
01-18 16:06:20.195   825 14215 I audio_hw_aoc: Mode Ambient is already selected
01-18 16:06:20.195   825 14215 D audio_route: Apply path: raw-playbackP
01-18 16:06:20.195   825  1326 D audio_hw_battery_adaptive_audio_control: ProcessPlaybackTunerControl: update playback tuner 1
01-18 16:06:20.195   825  1326 D audio_hw_waves_cust_action: UpdateThrottleState: 2->1 T
01-18 16:06:20.195   825  1326 D audio_hw_waves_cust_action: UpdateThrottleState: update tuning!
01-18 16:06:20.196   825 14215 D audio_hw_aoc_route: raw-playbackP 1
01-18 16:06:20.196   825 14215 D audio_route: Apply path: speaker-post
01-18 16:06:20.196   825 14215 D audio_hw_aoc_route: speaker-post 1
01-18 16:06:20.197   825 14215 D audio_hw_waves: update_rotation_from_adev: update sink 1 info
01-18 16:06:20.197   825 14215 D audio_hw_waves: update_rotation_from_adev: rotation update on:1, rotation:90
01-18 16:06:20.197   825 14215 D audio_hw_waves: set_waves_tuning_from_sink: initialize parameters successfully.
01-18 16:06:20.198   825 14215 D audio_hw_waves: send_waves_tuning: instance_id = 1, tuning_id = 2
01-18 16:06:20.198   825 14215 D audio_hw_aoc: aoc_get_hw_intf_ctrl : HW_INTF : 9
01-18 16:06:20.198   825 14215 D audio_hw_aoc: aoc_send_rtc_mixer_tuning: block_id = 16, comp_id = 2
01-18 16:06:20.198  1048  1048 D AOC     : F1:MSG: controller.cc, 273: AudioOutCtrl: cmd ID: 0x010f, tag: 0x9e [cntr = 163]
01-18 16:06:20.198  1048  1048 D AOC     : F1:Source 9 mastered by sink 0
01-18 16:06:20.198  1048  1048 D AOC     : F1:AT Map:200 (LL:0), 1 EPs active, Power: Yes, Config: No
01-18 16:06:20.198  1048  1048 D AOC     : F1:Mixer 0 configuration changed: (200)
01-18 16:06:20.198  1048  1048 D AOC     : F1:Sink 0 Configuration changed: ULL (enabled)
01-18 16:06:20.198  1048  1048 D AOC     : F1:AHWSinkSPKR started: 48 samples (32-bit,2 ch,48 kHz) block 384
01-18 16:06:20.198  1048  1048 D AOC     : F1:[AHWSinkSPKR] DL resync B: 0, A:0. wo: 0, target offset: 3264 (align 0) -> ro:0
01-18 16:06:20.198  1048  1048 D AOC     : F1:MSG: controller.cc, 811: AudioOutCtrl: ipc: audio_output_co, cmd ID: 0x010f, tag: 0x9e, rc: 0
01-18 16:06:20.198  1048  1048 D AOC     : F1:Speaker Started (DualDMA on 1 and 2)
01-18 16:06:20.198  1048  1048 D AOC     : H0:Mixer AMixSPKR: 480 samples (0200/0200) (OFF)
01-18 16:06:20.198  1048  1048 D AOC     : F1:MSG: controller.cc, 273: AudioOutCtrl: cmd ID: 0x00ce, tag: 0x9f [cntr = 164]
01-18 16:06:20.198  1048  1048 D AOC     : F1:ERR: controller_audio_output.cc, 1658: Entrypoint 23 cannot be configured
01-18 16:06:20.198  1048  1048 D AOC     : F1:ERR: controller.cc, 808: AudioOutCtrl: ipc: audio_output_co, cmd ID: 0x00ce, tag: 0x9f, rc: -1
01-18 16:06:20.198  1048  1048 D AOC     : F1:MSG: controller.cc, 273: AudioOutCtrl: cmd ID: 0x00d5, tag: 0x12 [cntr = 165]
01-18 16:06:20.200  1048  1048 D AOC     : H0:MSG: controller.cc, 811: AudOutCtrlH0: ipc: C-AO-H0, cmd ID: 0x00d5, tag: 0x20, rc: 0
01-18 16:06:20.200   825  1314 W audio_hw_stuning: receive_response_data:invalid rx_sz 0
01-18 16:06:20.200  1048  1048 D AOC     : F1:MSG: controller.cc, 811: AudioOutCtrl: ipc: audio_output_tu, cmd ID: 0x00d5, tag: 0x20, rc: 0
01-18 16:06:20.204  1048  1048 D AOC     : F1:MSG: controller.cc, 273: AudioOutCtrl: cmd ID: 0x00c9, tag: 0xa0 [cntr = 166]
01-18 16:06:20.204  1048  1048 D AOC     : F1:ERR: controller_audio_output.cc, 1480: Entrypoint 23 cannot be enabled
01-18 16:06:20.204  1048  1048 D AOC     : F1:ERR: controller.cc, 808: AudioOutCtrl: ipc: audio_output_co, cmd ID: 0x00c9, tag: 0xa0, rc: -1
01-18 16:06:20.204  1391  3195 W JobScheduler: Job didn't exist in JobStore: 90d0cde #u0a290/55 com.devexpert.weather/.controller.UpdateService
01-18 16:06:20.210  1049  1220 I CHRE    : @ 13536.172: [AR] inconsistent: 41.176471
01-18 16:06:20.211  1049  1220 D CHRE    : @ 13536.172: [ActivityPlatform] type 6, confidence 41
01-18 16:06:20.231 11665 11739 D TrafficStats: tagSocket(140) with statsTag=0xffffffff, statsUid=-1
01-18 16:06:20.372  1049  1220 I CHRE    : @ 13536.334: [ip] Peak detected: magnitude = 0.160893 hpa, duration = 80.000001 ms
01-18 16:06:20.372   835   842 D ContextHubHal: Got message from nanoapp: ID 0x476f6f676c001022
01-18 16:06:20.373   862  1512 I suez-nanoapp-clients: Barometric peak detected: magnitude = 0.160893 hPa (bucket: 4), duration = 80.000000 ms (bucket: 7).
01-18 16:06:20.373  1391  1773 E ContextHubClientManager: Cannot send message to unregistered client (host endpoint ID = -28638)
01-18 16:06:20.373   862  1512 I suez-nanoapp-clients: Vendor atom [id = 100047] reported.
```

The push notifications are enabled at this point, judging from callbacks from other attached system (facebook SDK and deep links).

Braze also updates my push notification key:

```
01-18 16:06:44.375 11665 11762 I Braze v23.3.0 .Braze: Requesting immediate data flush to Braze.
01-18 16:06:44.376 11665 11762 D Braze v23.3.0 .z0: bo.app.o0 fired:             commandType = ADD_REQUEST
01-18 16:06:44.376 11665 11762 D Braze v23.3.0 .z0:             brazeEvent = null
01-18 16:06:44.376 11665 11762 D Braze v23.3.0 .z0:             sessionId = null
01-18 16:06:44.376 11665 11762 D Braze v23.3.0 .z0:             brazeRequest = {
01-18 16:06:44.376 11665 11762 D Braze v23.3.0 .z0:   "respond_with": {
01-18 16:06:44.376 11665 11762 D Braze v23.3.0 .z0:     "user_id": "CORRECT_USER_ID",
01-18 16:06:44.376 11665 11762 D Braze v23.3.0 .z0:     "config": {
01-18 16:06:44.376 11665 11762 D Braze v23.3.0 .z0:       "config_time": 1656360902
01-18 16:06:44.376 11665 11762 D Braze v23.3.0 .z0:     }
01-18 16:06:44.376 11665 11762 D Braze v23.3.0 .z0:   }
01-18 16:06:44.376 11665 11762 D Braze v23.3.0 .z0: }
01-18 16:06:44.376 11665 11762 D Braze v23.3.0 .z0: Triggering bo.app.o0 on 1 subscribers.
01-18 16:06:44.376 11665 11762 V Braze v23.3.0 .q0: Added request to dispatcher with parameters: 
01-18 16:06:44.376 11665 11762 V Braze v23.3.0 .q0: {
01-18 16:06:44.376 11665 11762 V Braze v23.3.0 .q0:   "respond_with": {
01-18 16:06:44.376 11665 11762 V Braze v23.3.0 .q0:     "user_id": "CORRECT_USER_ID",
01-18 16:06:44.376 11665 11762 V Braze v23.3.0 .q0:     "config": {
01-18 16:06:44.376 11665 11762 V Braze v23.3.0 .q0:       "config_time": 1656360902
01-18 16:06:44.376 11665 11762 V Braze v23.3.0 .q0:     }
01-18 16:06:44.376 11665 11762 V Braze v23.3.0 .q0:   }
01-18 16:06:44.376 11665 11762 V Braze v23.3.0 .q0: }
01-18 16:06:44.376 11665 11758 V Braze v23.3.0 .q0: Adding SDK Auth token to request 'null'
01-18 16:06:44.381 11665 11758 D Braze v23.3.0 .k0: Cache locked successfully for export: bo.app.k0@a02dccc
01-18 16:06:44.381 11665 11758 V Braze v23.3.0 .k0: Sending full device due to NOTIFICATIONS_ENABLED true
01-18 16:06:44.382 11665 11758 V Braze v23.3.0 .t6: Push token cache cleared.
01-18 16:06:44.382 11665 11758 D Braze v23.3.0 .t6: Cache locked successfully for export: bo.app.t6@4465b15
01-18 16:06:44.388 11665 11809 D Braze v23.3.0 .m3: Making request with id => "4dc26fe6476b304c"
01-18 16:06:44.388 11665 11809 D Braze v23.3.0 .m3: to url: CORRECT_BRAZE_URL
01-18 16:06:44.388 11665 11809 D Braze v23.3.0 .m3:             
01-18 16:06:44.388 11665 11809 D Braze v23.3.0 .m3: with headers:
01-18 16:06:44.388 11665 11809 D Braze v23.3.0 .m3: "Accept-Encoding" => "gzip, deflate"
01-18 16:06:44.388 11665 11809 D Braze v23.3.0 .m3: "Content-Type" => "application/json"
01-18 16:06:44.388 11665 11809 D Braze v23.3.0 .m3: "X-Braze-Api-Key" => "CORRECT_API_KEY"
01-18 16:06:44.388 11665 11809 D Braze v23.3.0 .m3: 
01-18 16:06:44.388 11665 11809 D Braze v23.3.0 .m3: and JSON :
01-18 16:06:44.388 11665 11809 D Braze v23.3.0 .m3: {
01-18 16:06:44.388 11665 11809 D Braze v23.3.0 .m3:   "device_id": "CORRECT_DEVICE_ID",
01-18 16:06:44.388 11665 11809 D Braze v23.3.0 .m3:   "time": 1674076004,
01-18 16:06:44.388 11665 11809 D Braze v23.3.0 .m3:   "api_key": "CORRECT_API_KEY",
01-18 16:06:44.388 11665 11809 D Braze v23.3.0 .m3:   "sdk_version": "23.3.0",
01-18 16:06:44.388 11665 11809 D Braze v23.3.0 .m3:   "app_version": "1.3.0",
01-18 16:06:44.388 11665 11809 D Braze v23.3.0 .m3:   "app_version_code": "1.0.0.0",
01-18 16:06:44.388 11665 11809 D Braze v23.3.0 .m3:   "device": {
01-18 16:06:44.388 11665 11809 D Braze v23.3.0 .m3:     "os_version": "33",
01-18 16:06:44.388 11665 11809 D Braze v23.3.0 .m3:     "carrier": "Freedom",
01-18 16:06:44.388 11665 11809 D Braze v23.3.0 .m3:     "model": "Pixel 7",
01-18 16:06:44.388 11665 11809 D Braze v23.3.0 .m3:     "resolution": "1080x2400",
01-18 16:06:44.388 11665 11809 D Braze v23.3.0 .m3:     "locale": "en_US",
01-18 16:06:44.388 11665 11809 D Braze v23.3.0 .m3:     "remote_notification_enabled": true,
01-18 16:06:44.388 11665 11809 D Braze v23.3.0 .m3:     "android_is_background_restricted": false,
01-18 16:06:44.388 11665 11809 D Braze v23.3.0 .m3:     "time_zone": "America\/Toronto"
01-18 16:06:44.388 11665 11809 D Braze v23.3.0 .m3:   },
01-18 16:06:44.388 11665 11809 D Braze v23.3.0 .m3:   "attributes": [
01-18 16:06:44.388 11665 11809 D Braze v23.3.0 .m3:     {
01-18 16:06:44.388 11665 11809 D Braze v23.3.0 .m3:       "push_subscribe": "opted_in",
01-18 16:06:44.388 11665 11809 D Braze v23.3.0 .m3:       "custom": {
01-18 16:06:44.388 11665 11809 D Braze v23.3.0 .m3:         "player_level": 106
01-18 16:06:44.388 11665 11809 D Braze v23.3.0 .m3:       },
01-18 16:06:44.388 11665 11809 D Braze v23.3.0 .m3:       "user_id": "CORRECT_USER_ID",
01-18 16:06:44.388 11665 11809 D Braze v23.3.0 .m3:       "push_token": "CORRECT_TOKEN"
01-18 16:06:44.388 11665 11809 D Braze v23.3.0 .m3:     }
01-18 16:06:44.388 11665 11809 D Braze v23.3.0 .m3:   ],
01-18 16:06:44.388 11665 11809 D Braze v23.3.0 .m3:   "sdk_flavor": "unity",
01-18 16:06:44.388 11665 11809 D Braze v23.3.0 .m3:   "respond_with": {
01-18 16:06:44.388 11665 11809 D Braze v23.3.0 .m3:     "user_id": "CORRECT_USER_ID",
01-18 16:06:44.388 11665 11809 D Braze v23.3.0 .m3:     "config": {
01-18 16:06:44.388 11665 11809 D Braze v23.3.0 .m3:       "config_time": 1656360902
01-18 16:06:44.388 11665 11809 D Braze v23.3.0 .m3:     }
01-18 16:06:44.388 11665 11809 D Braze v23.3.0 .m3:   }
01-18 16:06:44.388 11665 11809 D Braze v23.3.0 .m3: }
01-18 16:06:44.472 11665 11809 D TrafficStats: tagSocket(83) with statsTag=0x539, statsUid=-1
01-18 16:06:44.509 11665 11809 V Braze v23.3.0 .v5: Enabling SSL protocols: [TLSv1, TLSv1.1, TLSv1.2, TLSv1.3]
01-18 16:06:44.609 11665 11809 D Braze v23.3.0 .m3: Result(id = 4dc26fe6476b304c time = 221ms)
01-18 16:06:44.609 11665 11809 D Braze v23.3.0 .m3: {}
01-18 16:06:44.609 11665 11809 D Braze v23.3.0 .bo.app.m6: Triggered actions Json array was null. Not de-serializing triggered actions.
01-18 16:06:44.609 11665 11809 D Braze v23.3.0 .bo.app.m6: Templated message Json was null. Not de-serializing templated message.
01-18 16:06:44.609 11665 11809 D Braze v23.3.0 .i0: DataSyncRequest executed successfully.
01-18 16:06:44.609 11665 11809 V Braze v23.3.0 .s: Processing server response payload for user with id: 45
01-18 16:06:44.611 11665 11809 D Braze v23.3.0 .z0: bo.app.u4 fired: RequestNetworkSuccessEvent(request={
01-18 16:06:44.611 11665 11809 D Braze v23.3.0 .z0:   "device_id": "CORRECT_DEVICE_ID",
01-18 16:06:44.611 11665 11809 D Braze v23.3.0 .z0:   "time": 1674076004,
01-18 16:06:44.611 11665 11809 D Braze v23.3.0 .z0:   "api_key": "CORRECT_API_KEY",
01-18 16:06:44.611 11665 11809 D Braze v23.3.0 .z0:   "sdk_version": "23.3.0",
01-18 16:06:44.611 11665 11809 D Braze v23.3.0 .z0:   "app_version": "1.3.0",
01-18 16:06:44.611 11665 11809 D Braze v23.3.0 .z0:   "app_version_code": "1.0.0.0",
01-18 16:06:44.611 11665 11809 D Braze v23.3.0 .z0:   "device": {
01-18 16:06:44.611 11665 11809 D Braze v23.3.0 .z0:     "os_version": "33",
01-18 16:06:44.611 11665 11809 D Braze v23.3.0 .z0:     "carrier": "Freedom",
01-18 16:06:44.611 11665 11809 D Braze v23.3.0 .z0:     "model": "Pixel 7",
01-18 16:06:44.611 11665 11809 D Braze v23.3.0 .z0:     "resolution": "1080x2400",
01-18 16:06:44.611 11665 11809 D Braze v23.3.0 .z0:     "locale": "en_US",
01-18 16:06:44.611 11665 11809 D Braze v23.3.0 .z0:     "remote_notification_enabled": true,
01-18 16:06:44.611 11665 11809 D Braze v23.3.0 .z0:     "android_is_background_restricted": false,
01-18 16:06:44.611 11665 11809 D Braze v23.3.0 .z0:     "time_zone": "America\/Toronto"
01-18 16:06:44.611 11665 11809 D Braze v23.3.0 .z0:   },
01-18 16:06:44.611 11665 11809 D Braze v23.3.0 .z0:   "attributes": [
01-18 16:06:44.611 11665 11809 D Braze v23.3.0 .z0:     {
01-18 16:06:44.611 11665 11809 D Braze v23.3.0 .z0:       "push_subscribe": "opted_in",
01-18 16:06:44.611 11665 11809 D Braze v23.3.0 .z0:       "custom": {
01-18 16:06:44.611 11665 11809 D Braze v23.3.0 .z0:         "player_level": 106
01-18 16:06:44.611 11665 11809 D Braze v23.3.0 .z0:       },
01-18 16:06:44.611 11665 11809 D Braze v23.3.0 .z0:       "user_id": "CORRECT_USER_ID",
01-18 16:06:44.611 11665 11809 D Braze v23.3.0 .z0:       "push_token": "CORRECT_TOKEN"
01-18 16:06:44.611 11665 11809 D Braze v23.3.0 .z0:     }
01-18 16:06:44.611 11665 11809 D Braze v23.3.0 .z0:   ],
01-18 16:06:44.611 11665 11809 D Braze v23.3.0 .z0:   "sdk_flavor": "unity",
01-18 16:06:44.611 11665 11809 D Braze v23.3.0 .z0:   "respond_with": {
01-18 16:06:44.611 11665 11809 D Braze v23.3.0 .z0:     "user_id": "CORRECT_USER_ID",
01-18 16:06:44.611 11665 11809 D Braze v23.3.0 .z0:     "config": {
01-18 16:06:44.611 11665 11809 D Braze v23.3.0 .z0:       "config_time": 1656360902
01-18 16:06:44.611 11665 11809 D Braze v23.3.0 .z0:     }
01-18 16:06:44.611 11665 11809 D Braze v23.3.0 .z0:   }
01-18 16:06:44.611 11665 11809 D Braze v23.3.0 .z0: })
01-18 16:06:44.611 11665 11809 D Braze v23.3.0 .z0: Triggering bo.app.u4 on 1 subscribers.
01-18 16:06:44.612 11665 11809 D Braze v23.3.0 .z0: bo.app.r0 fired: DispatchSucceededEvent(request={
01-18 16:06:44.612 11665 11809 D Braze v23.3.0 .z0:   "device_id": "CORRECT_DEVICE_ID",
01-18 16:06:44.612 11665 11809 D Braze v23.3.0 .z0:   "time": 1674076004,
01-18 16:06:44.612 11665 11809 D Braze v23.3.0 .z0:   "api_key": "CORRECT_API_KEY",
01-18 16:06:44.612 11665 11809 D Braze v23.3.0 .z0:   "sdk_version": "23.3.0",
01-18 16:06:44.612 11665 11809 D Braze v23.3.0 .z0:   "app_version": "1.3.0",
01-18 16:06:44.612 11665 11809 D Braze v23.3.0 .z0:   "app_version_code": "1.0.0.0",
01-18 16:06:44.612 11665 11809 D Braze v23.3.0 .z0:   "device": {
01-18 16:06:44.612 11665 11809 D Braze v23.3.0 .z0:     "os_version": "33",
01-18 16:06:44.612 11665 11809 D Braze v23.3.0 .z0:     "carrier": "Freedom",
01-18 16:06:44.612 11665 11809 D Braze v23.3.0 .z0:     "model": "Pixel 7",
01-18 16:06:44.612 11665 11809 D Braze v23.3.0 .z0:     "resolution": "1080x2400",
01-18 16:06:44.612 11665 11809 D Braze v23.3.0 .z0:     "locale": "en_US",
01-18 16:06:44.612 11665 11809 D Braze v23.3.0 .z0:     "remote_notification_enabled": true,
01-18 16:06:44.612 11665 11809 D Braze v23.3.0 .z0:     "android_is_background_restricted": false,
01-18 16:06:44.612 11665 11809 D Braze v23.3.0 .z0:     "time_zone": "America\/Toronto"
01-18 16:06:44.612 11665 11809 D Braze v23.3.0 .z0:   },
01-18 16:06:44.612 11665 11809 D Braze v23.3.0 .z0:   "attributes": [
01-18 16:06:44.612 11665 11809 D Braze v23.3.0 .z0:     {
01-18 16:06:44.612 11665 11809 D Braze v23.3.0 .z0:       "push_subscribe": "opted_in",
01-18 16:06:44.612 11665 11809 D Braze v23.3.0 .z0:       "custom": {
01-18 16:06:44.612 11665 11809 D Braze v23.3.0 .z0:         "player_level": 106
01-18 16:06:44.612 11665 11809 D Braze v23.3.0 .z0:       },
01-18 16:06:44.612 11665 11809 D Braze v23.3.0 .z0:       "user_id": "CORRECT_USER_ID",
01-18 16:06:44.612 11665 11809 D Braze v23.3.0 .z0:       "push_token": "CORRECT_TOKEN"
01-18 16:06:44.612 11665 11809 D Braze v23.3.0 .z0:     }
01-18 16:06:44.612 11665 11809 D Braze v23.3.0 .z0:   ],
01-18 16:06:44.612 11665 11809 D Braze v23.3.0 .z0:   "sdk_flavor": "unity",
01-18 16:06:44.612 11665 11809 D Braze v23.3.0 .z0:   "respond_with": {
01-18 16:06:44.612 11665 11809 D Braze v23.3.0 .z0:     "user_id": "CORRECT_USER_ID",
01-18 16:06:44.612 11665 11809 D Braze v23.3.0 .z0:     "config": {
01-18 16:06:44.612 11665 11809 D Braze v23.3.0 .z0:       "config_time": 1656360902
01-18 16:06:44.612 11665 11809 D Braze v23.3.0 .z0:     }
01-18 16:06:44.612 11665 11809 D Braze v23.3.0 .z0:   }
01-18 16:06:44.612 11665 11809 D Braze v23.3.0 .z0: })
01-18 16:06:44.612 11665 11809 D Braze v23.3.0 .z0: Triggering bo.app.r0 on 1 subscribers.
01-18 16:06:44.613 11665 11809 V Braze v23.3.0 .k0: Notifying confirmAndUnlock listeners for cache: bo.app.k0@a02dccc
01-18 16:06:44.613 11665 11809 V Braze v23.3.0 .t6: Notifying confirmAndUnlock listeners for cache: bo.app.t6@4465b15
01-18 16:06:44.614 11665 11809 V Braze v23.3.0 .e5: Attempting to unlock server config info.
01-18 16:06:44.614 11665 11809 D Braze v23.3.0 .e5: Unlocking config info lock.
```

(I have excised some game/company identifying IDs)

3.

See attached image. Target API is 33 (to allow for push prompt). Currently the push prompt does appear as expected but no callback happens.

Building with Unity 2021.3.16f1 (the current LTS release).

![Screen Shot 2023-01-18 at 3 46 22 PM](https://user-images.githubusercontent.com/1449818/213290953-4e4b3d5e-f3eb-4af9-a3f3-7eedfec77fa9.png)

**Comment by radixdev** [comment]
Ah I see. This is expected behavior for Android Unity. That callback is only fired on iOS, apologies for the confusion. The core Braze Android SDK does not offer a way to know the prompt return value and as a result, this functionality is not possible on Braze Android Unity either.

**Comment by indiecore** [comment]
Ah, I logged this as a feature request, not a bug report.

**Comment by radixdev** [comment]
In that case, I would recommend raising this through your CSM or support@braze.com instead. I misread this as a bug report, sorry about that.
